### PR TITLE
refactor: add option for removable tag for selectable lists

### PR DESCRIPTION
**Pull request by D-B-Hawk**
removable tag

### CODE REVIEW

File: components/tag/index.tsx [hunk 1]
```diff
@@ -33,19 +36,53 @@ export const TAG_ICON_OPTONS = {
...
-);
+const Tag: FunctionComponent<TagProps> = ({
+  text,
+  icon,
+  removable,
+  bgColor,
+  onRemoval = noop,
+  ...rest
+}) => {
+  const { bg, textColor } = TAG_COLOR_MAP[bgColor ?? 'none'];
```
Comment (CristhianF7) [inline components/tag/index.tsx:74]: can we add a fallback here please?

Comment (D-B-Hawk) [reply components/tag/index.tsx:74]: the fallback is "none" in this case.

Comment (CristhianF7) [reply components/tag/index.tsx:74]: that only covers when there is no `bgColor`.

TAG_COLOR_MAP[bgColor ?? 'none'] || TAG_COLOR_MAP['none']

Comment (D-B-Hawk) [reply components/tag/index.tsx:74]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Comment (D-B-Hawk) [reply components/tag/index.tsx:74]: i see what you are driving at. you want a fallback for an invalid color.

**Review by CristhianF7 — APPROVED** [review]

**Pull request merged**